src/openrct2-ui/interface/Widget.cpp

```diff
@@ -288,7 +289,8 @@ namespace OpenRCT2::Ui
         // Check if the button is pressed down
         if (WidgetIsPressed(w, widgetIndex) || isToolActive(w, widgetIndex))
         {
-            if (static_cast<int32_t>(widget.image.ToUInt32()) == -2)
+            // Dead code?
+            if (static_cast<int32_t>(widget.image.GetIndex()) == -2)
```

Comment:
I don't think can happens any more since #22018. Thoughts?

Reply:
Could add a Guard::Assert here for a while if unsure, I think its best left alone for now.